packages/core/src/browser/shell/application-shell.ts

```diff
@@ -947,7 +947,7 @@ export class ApplicationShell extends Widget {
      * Set the z-index of the given element and its ancestors to the value `z`.
      */
     private setZIndex(element: HTMLElement, z: string | null): void {
-        element.style.zIndex = z;
+        element.style.zIndex = z || '';
```

Comment:
Default value is [`auto`](https://www.w3schools.com/cssref/pr_pos_z-index.asp), should we put that?

Reply:
'' tells the browser remove the property, and browser will decide what is the default

Reply:
Ok, did not know that.